src/librepgp/stream-common.cpp

```diff
@@ -892,7 +892,8 @@ init_tmpfile_dest(pgp_dest_t *dst, const char *path, bool overwrite)
         RNP_LOG("failed to build file path");
         return RNP_ERROR_BAD_PARAMETERS;
     }
-    mktemp(tmp);
+    close(mkstemp(tmp));
+    unlink(tmp);
```

Comment:
Why did you need to introduce these changes? This lines doesn't seem to produce any warning (GCC 9.2.0).